components/automate-chef-io/data/docs/api-static/05-node-export.swagger.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This is the main change. Manually created the swagger doc for the streaming end points.